lib/usd/ui/USDImportDialog.ui

```diff
@@ -23,7 +23,75 @@
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
+   <item row="0" column="0">
+    <widget class="QMenuBar" name="menuBar">
+     <widget class="QMenu" name="menuEdit">
+      <property name="title">
+       <string>Edit</string>
+      </property>
+      <addaction name="actionReset_File"/>
+     </widget>
+     <addaction name="menuEdit"/>
+    </widget>
+   </item>   
+   <item row="1" column="0">
+    <layout class="QHBoxLayout" name="horizontalLayout_4">
+     <property name="topMargin">
+      <number>7</number>
+     </property>
+     <property name="bottomMargin">
+      <number>7</number>
+     </property>
+     <item>
+      <widget class="QLabel" name="label">
+       <property name="autoFillBackground">
+        <bool>true</bool>
+       </property>
+       <property name="text">
+        <string>USD File Path:</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QLineEdit" name="usdFilePath">
+       <property name="enabled">
+        <bool>false</bool>
+       </property>
+       <property name="readOnly">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+    </layout>
+   </item>
+   <item row="2" column="0">
+    <widget class="QLabel" name="selectPrims">
+     <property name="text">
+      <string>Select the prim(s) you'd like to import. Switch variants accordingly.</string>
+     </property>
+    </widget>
+   </item>   
    <item row="3" column="0">
+    <layout class="QVBoxLayout" name="verticalLayout">
+     <item>
+      <widget class="QTreeView" name="treeView">
+       <property name="autoFillBackground">
+        <bool>true</bool>
+       </property>
+       <property name="editTriggers">
+        <set>QAbstractItemView::NoEditTriggers</set>
+       </property>
+       <property name="selectionMode">
+        <enum>QAbstractItemView::ExtendedSelection</enum>
+       </property>
+       <property name="uniformRowHeights">
+        <bool>false</bool>
+       </property>
+      </widget>
+     </item>
+    </layout>
+   </item>
+   <item row="4" column="0">
```

Comment:
.ui file will be hard to review, but the reason it looks like so much has changed in the diff is because the new menu item goes at the top, so it becomes row 0.  This pushes all of the other items down a row and ends up creating a diff like this.

I found it harder than expected to add the menu bar + item at the top of the dialog, there is no interactive way of doing this in Qt Designer as far as I can tell unless we change the base class type of the dialog to a window.  I did find a way to add the menu bar as a widget to a layout and ended up basically hand-editing this file to make it work.

Reply:
Okay as long as it still works. Sometimes even when editing in Qt Designer and adding one thing the diff was all messed up.